anisoap/representations/ellipsoidal_density_projection.py

```diff
@@ -129,7 +133,11 @@ def pairwise_ellip_expansion(
                         deg = l + 2 * (num_ns[l] - 1)
                         moments_l = moments[: deg + 1, : deg + 1, : deg + 1]
                         values_ldict[l].append(
-                            np.einsum("mnpqr, pqr->mn", sph_to_cart[l], moments_l)
+                            np.einsum(
+                                "mnpqr, pqr->mn",
+                                sph_to_cart[l] / solid_harm_prefact[l],
+                                moments_l,
+                            )
```

Comment:
Are we adding the matrix-matrix multiplication between the coefficients and moments? Why do we compute and add this?

Reply:
This piece of code combines the moments and the cartesian representation of the spherical harmonics to obtain each coefficient that is present in descriptor. It is replicating equation A30 (in the appendix) in the [AniSOAP paper](https://arxiv.org/abs/2403.19039)